src/components/List/index.jsx

```diff
@@ -39,10 +39,28 @@ export const Header = ({ children }) => (
   <div className={styles['c-list-header']}>{children}</div>
 )
 
-const _Row = ({ className, onRef, ...rest }) => (
-  <div ref={onRef} className={cx(styles['c-list-row'], className)} {...rest} />
+const _Row = ({ className, onRef, active, ...rest }) => (
+  <div
+    ref={onRef}
+    className={cx(
+      styles['c-list-row'],
+      {
+        [styles['disable-active']]: !active
+      },
+      className
+    )}
+    {...rest}
+  />
 )
 
+_Row.defaultProps = {
+  active: true
+}
+
+_Row.propTypes = {
+  active: PropTypes.bool
+}
```

Comment:
Instead of improving our adhoc/custom list, I think we should use MUI/cozy-uis's List / ListItem. The `button` props of ListItems controls whether the ListItem is clickable.

Reply:
Ok thanks for the information.

It's changed